packages/select/src/MenuButton.tsx

```diff
@@ -228,12 +228,18 @@ interface MenuButtonProps
   >;
 }
 
+type LabelProp = Pick<
```

Comment:
this should probably live somewhere communal at this point -- maybe in the a11y package?

Reply:
I thought about this, but I think Select is one of those components that's overdue a major refactor, so I didn't want to touch it up too much. I also feel like this is pretty relevant to the form control / input base component work we'll be doing in Q1, so we'll be doing a deeper dive on how to make cross-package changes soon anyways.